src/AspNet.Security.OAuth.Strava/StravaAuthenticationHandler.cs

```diff
@@ -60,5 +61,7 @@ public class StravaAuthenticationHandler : OAuthHandler<StravaAuthenticationOpti
             await Options.Events.CreatingTicket(context);
             return new AuthenticationTicket(context.Principal, context.Properties, Scheme.Name);
         }
+
+        protected override string FormatScope() => !Options.Scope.Any() ? "public" : string.Join(",", Options.Scope);
```

Comment:
The `string.Join(",", Options.Scope)` part is the right thing to do, but I'm not sure I like the first part (i.e only adding "public" is no explicit scope as manually added). I'll think more about it and come back to you.

Reply:
Was unsure about that one myself. The reason I added it was to maintain backwards compatibility since "public" is no longer set by default. I'll be happy to remove it, though.